scripts/linux/appimage/soh.sh.in

```diff
@@ -23,7 +23,7 @@ while [[ (! -e "$SHIP_HOME"/oot.otr) || (! -e "$SHIP_HOME"/oot-mq.otr) ]]; do
         do
             if [[ -e $romfile ]]; then
                 export ASSETDIR="$(mktemp -d /tmp/assets-XXXXX)"
-                ln -s "$HERE"/usr/bin/{assets,soh.elf,ZAPD} "$ASSETDIR"
+                ln -s "$SHIP_BIN_DIR"/{assets,soh.elf,ZAPD} "$ASSETDIR"
```

Comment:
https://github.com/HarbourMasters/Shipwright/blob/5b81964ea5c0c1087f20b4f8da5a5605501784e8/scripts/linux/appimage/soh.sh.in#L13

LGTM